guides/v2.3/graphql/reference/wishlist.md

```diff
@@ -0,0 +1,94 @@
+---
+group: graphql
+title: Wish list endpoint
+---
+
+The Wish list endpoint returns the contents of a customer's wish list.
```

Comment:
```suggestion
The Wish list endpoint defines the contents of a customer's wish list.
```

Reply:
Create a "Query" H2 and a bridge sentence describing the query.

Reply:
done